src/data/battler-tags.ts

```diff
@@ -2109,6 +2114,48 @@ export class CritBoostTag extends BattlerTag {
   }
 }
 
+/**
+ * Used for an instance of a stackable crit boost
+ * Granted by G-Max Chi Strike
+ */
+export class ChiStrikeCritBoostTag extends BattlerTag {
+  public stackCount: number = 0;
```

Comment:
why public?

Reply:
I also wonder, if this is safe to be a `number` or must be a `NumberHolder`. I'm unfortunately not able to tell